### PR TITLE
Updated Windows image sysprep for image acctests

### PR DESCRIPTION
**Pull request by jackofallops**

### CODE REVIEW

**Comment by jackofallops** [comment]
The travis build failure's unrelated and has been fixed via #5669

File: azurerm/internal/services/compute/tests/resource_arm_image_test.go [hunk 1]
```diff
@@ -258,6 +260,50 @@ func TestAccAzureRMImageVMSS_customImageVMSSFromVHD(t *testing.T) {
...
+func testGeneralizeWindowsVMImage(resourceGroup string, name string, user string, password string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		client := acceptance.AzureProvider.Meta().(*clients.Client).Compute.VMClient
+		ctx := acceptance.AzureProvider.Meta().(*clients.Client).StopContext
+
+		//command := []string{"%windir%\\system32\\sysprep\\Sysprep.exe /oobe /generalize /shutdown /mode:vm /quiet"}
+		command := []string{
+			"$cmd = \"$Env:SystemRoot\\system32\\sysprep\\sysprep.exe\"",
+			"$args = \"/generalize /oobe /mode:vm /quiet\"",
+			//"Start-Process -Filepath $cmd -ArgumentList $args -Wait",
```
Comment (tombuildsstuff) [inline azurerm/internal/services/compute/tests/resource_arm_image_test.go]:
```suggestion
```

File: azurerm/internal/services/compute/tests/resource_arm_image_test.go [hunk 2]
```diff
@@ -258,6 +260,50 @@ func TestAccAzureRMImageVMSS_customImageVMSSFromVHD(t *testing.T) {
 	})
 }
 
+func testGeneralizeWindowsVMImage(resourceGroup string, name string, user string, password string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		client := acceptance.AzureProvider.Meta().(*clients.Client).Compute.VMClient
+		ctx := acceptance.AzureProvider.Meta().(*clients.Client).StopContext
+
+		//command := []string{"%windir%\\system32\\sysprep\\Sysprep.exe /oobe /generalize /shutdown /mode:vm /quiet"}
```
Comment (tombuildsstuff) [inline azurerm/internal/services/compute/tests/resource_arm_image_test.go]:
```suggestion
```

File: azurerm/internal/services/compute/tests/windows_virtual_machine_resource_images_test.go [hunk 3]
```diff
@@ -394,11 +394,9 @@ func generalizeWindowsVirtualMachine(resourceName string) func(s *terraform.Stat
 		// TODO: make this for Windows..
```
Comment (tombuildsstuff) [inline azurerm/internal/services/compute/tests/windows_virtual_machine_resource_images_test.go]:
```suggestion
```

**Pull request merged**